interactive_engine/common/pom.xml

```diff
@@ -64,6 +64,10 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+    </dependency>
```

Comment:
We already have jackson as json parser. I think there is no need to introduce another depencency.